dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -462,11 +455,53 @@ export const AdSlot = ({
 					);
 				}
 				case ArticleDisplay.Standard: {
+					const slotId = 'dfp-ad--right';
 					return (
-						<TopRightAdSlot
-							isPaidContent={isPaidContent}
-							adStyles={[labelStyles]}
-						/>
+						<>
+							<Island
+								priority="enhancement"
```

Comment:
Is this an “enhancement” or a “feature”? Worth noting that [for the small number of pages that are slow enough to be adapted](https://github.com/guardian/dotcom-rendering/pull/11174), enhancements will not be run.

Reply:
Didn't think about this too deeply... I think perhaps a `feature` would be wiser.

Reply:
Just a note to say that actually, [“feature” will also be switched off when adapted](https://github.com/guardian/dotcom-rendering/blob/670a45e27f9b4b8b105058286fe8e09026417d52/dotcom-rendering/src/client/adaptiveSite.ts#L38-L41).